code/app/src/main/java/com/example/the_tarlords/ui/event/EventDetailsFragment.java

```diff
@@ -178,6 +179,21 @@ public void onClick(View v) {
                 qrcode.shareQR(imageView, getActivity());
             }
         });
+        // On-click Listener for the map image
+        eventMapImageView.setOnClickListener(new View.OnClickListener() {
```

Comment:
Is this going to be viewable all the time? Because I don't think that the attendees should be able to see where other people have checked in

Reply:
I'm not sure how Rimsha and Lucy have the xml set up, should I put it under the if(organizer) {.....}